packages/kas/src/types.d.ts

```diff
@@ -17,6 +17,7 @@ export type ExpressionVars = {
     equalIgnoringCase: boolean;
 };
 
+// TODO: Convert to interface and make the various Expr's implement it?
 export type Expression = {
```

Comment:
All of the custom classes in `nodes.ts` conform to this type, but implicitly. That's not obvious.